packages/cookie-handler/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@goldinteractive/feature-cookie-disclaimer",
+  "version": "0.0.11",
+  "description": "cookie-disclaimer feature",
+  "main": "lib/main.min.js",
+  "module": "src/index.js",
+  "author": "Gold Interactive",
+  "license": "MIT",
+  "scripts": {
+    "build": "node ../../scripts/build.js",
+    "watch": "node ../../scripts/watch.js"
+  },
+  "dependencies": {
+    "js-cookie": "^2.2.0",
+    "tiny-invariant": "^1.0.6"
```

Comment:
Tiny Invariant shouldn't be added to dependencies - instead the sem version of `js-base` should be updated to `0.1.0` which includes `tiny-invariant`.